editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -216,12 +217,31 @@ function useFindValidTarget(): (
   })
 
   return React.useCallback(
-    (selectableViews: Array<ElementPath>, mousePoint: WindowPoint | null) => {
-      const { selectedViews } = storeRef.current
-      const validElementMouseOver: ElementPath | null = getValidTargetAtPoint(
-        selectableViews,
-        mousePoint,
-      )
+    (
+      selectableViews: Array<ElementPath>,
+      mousePoint: WindowPoint | null,
+      preferAlreadySelected: boolean,
```

Comment:
Can you change this to a union of strings please to aid readability of the function calls